docs/design/vsphereCredentials.md

```diff
@@ -0,0 +1,61 @@
+## Passing vsphere credentials
+For the cluster-api vsphere provider to work, the users need to provide the vsphere credentials to access the infrastructure. There are 2 ways how the users can provide these credentials.
+
+1. Using kubernetes `secrets`
+   1. Create a secret that contains 2 keys namely `username` and `password` in the same namespace as the desired `Cluster` object.
```

Comment:
Can you label this a or - instead of 1?  Markdown should allow you to do -.  It's easier to follow if we don't have two 1 and two 2.